app/controllers/projects_controller.rb

```diff
@@ -33,6 +33,9 @@ def create
     @project = Project.new(project_params)
 
     if @project.save
+      if ENV['PROJECT_CREATED_NOTIFY_ADDRESS']
+        ProjectMailer.created_email(@current_user,@project).deliver_later
```

Comment:
Yeah, I was just thinking about the future. Even the rails docs say use
_later unless you NEED to have it always delivered immediately.

On Thu, Jan 8, 2015 at 5:22 PM, Steven Davidovitz <notifications@github.com>
wrote:

> In app/controllers/projects_controller.rb
> <https://github.com/zendesk/samson/pull/223#discussion-diff-22696727>:
>
> > @@ -33,6 +33,9 @@ def create
> >      @project = Project.new(project_params)
> >
> >      if @project.save
> > +      if ENV['PROJECT_CREATED_NOTIFY_ADDRESS']
> > +        ProjectMailer.created_email(@current_user,@project).deliver_later
>
> We don't actually have a queue so this essentially runs inline anyway
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zendesk/samson/pull/223/files#r22696727>.
>